airbyte-webapp/src/hooks/services/useSourceHook.tsx

```diff
@@ -149,24 +148,23 @@ const useUpdateSource = () => {
   );
 };
 
+export type SchemaError = (Error & { status: number; response: SynchronousJobRead }) | null;
+
 const useDiscoverSchema = (
   sourceId: string,
   disableCache?: boolean
 ): {
   isLoading: boolean;
   schema: SyncSchema;
-  schemaErrorStatus: { status: number; response: SynchronousJobRead } | null;
+  schemaErrorStatus: SchemaError;
   catalogId: string | undefined;
   onDiscoverSchema: () => Promise<void>;
 } => {
   const service = useSourceService();
   const [schema, setSchema] = useState<SyncSchema>({ streams: [] });
   const [catalogId, setCatalogId] = useState<string | undefined>("");
   const [isLoading, setIsLoading] = useState(false);
-  const [schemaErrorStatus, setSchemaErrorStatus] = useState<{
-    status: number;
-    response: JobInfo;
```

Comment:
The type of this `response` field was `JobInfo` before your changes, and now it will be `SynchronousJobRead` due to the type you defined. Is this type change okay or could it break something?

Reply:
I'm not sure. Swapping these doesn't seem to cause a compile issue so I'll switch to `JobInfo`